packages/server/src/app/user/user.validator.js

```diff
@@ -1,3 +1,18 @@
+export const passwordValidator = {
+  notEmpty: { bail: true, errorMessage: 'Een wachtwoord is verplicht.' },
+  isStrongPassword: {
+    bail: true,
+    options: { minLength: 8, minLowercase: 1, minNumbers: 1, minSymbols: 1, minUppercase: 1 },
+    errorMessage:
+      'We raden aan om een combinatie te gebruiken van minimaal 1 hoofdletters, 1 kleine letters, 1 symbool, 1 cijfer en een minimaal lengte van 8 tekens.'
```

Comment:
'We raden aan' sounds optional, although it isn't

Reply:
I've simply re-used the logic that was already there, including the errorMessage, but you are right, so I'll change that to a more clearer message.